hack/validate/default

```diff
@@ -11,6 +11,7 @@ SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 . "${SCRIPTDIR}"/yamllint
 . "${SCRIPTDIR}"/swagger
 . "${SCRIPTDIR}"/swagger-gen
-. "${SCRIPTDIR}"/deprecate-integration-cli
+# FIXME(thaJeztah): re-enable deprecate-integration-cli once https://github.com/moby/moby/pull/42300 is merged
+#. "${SCRIPTDIR}"/deprecate-integration-cli
```

Comment:
Looking at: https://github.com/moby/moby/pull/42300/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88

I think we don't need to comment it out?

Reply:
👍 removed